frappe/utils/change_log.py

```diff
@@ -129,7 +129,7 @@ def get_app_branch(app):
 def get_app_last_commit_ref(app):
 	try:
 		result = subprocess.check_output('cd ../apps/{0} && git rev-parse HEAD --short 7'.format(app),
-			shell=True)
+			shell=True, stdin=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
subprocess.DEVNULL is in PY3... do not backport.
https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL